packages/base/room.gts

```diff
@@ -464,7 +467,7 @@ export class RoomField extends FieldDef {
         roomMemberCache.set(this, roomMembers);
       }
 
-      for (let event of this.newEvents) {
+      for (let event of this.events) {
```

Comment:
Solving similar issue than here (see the comment):

https://github.com/cardstack/boxel/blob/b7f0d4e02e2bafdf5d9fe7486c1a3cd63689022b/packages/base/room.gts#L389-L407